packages/eslint-plugin-twenty/src/rules/no-state-useref.ts

```diff
@@ -0,0 +1,68 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+
+const createRule = ESLintUtils.RuleCreator(() => `https://docs.twenty.com`);
+
+const noStateUseRef = createRule({
+  create: (context) => {
+    return {
```

Comment:
@gitstart-twenty we prefer this way of writing if condition that avoids making a complex conditions and enforce runtime typing properly